src/encoded/docs/schema-changes.md

```diff
@@ -154,69 +154,141 @@ This new object is an array of example objects that can successfully POST agains
+
+**Follow the steps as outlined below**
+
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. 
+
+**Specific example from the genetic modifications object schema change:**
+
+Up until schema version 6 for the genetic modifications object, one of the possible enums for "purpose" property was "validation". As a part of schema version upate to 7, "validation" was removed and instead the term "characterization" was added.
+
 
-1. In the **schemas** directory, increment the schema version (1->2), if it meets the above rule: 
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "validation",
+                "screening",
+                "expression"
+            ]
+        },
+        # Replacing the enum "validation" by the enum "characterization" in the list of enums of the "purpose" property:
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "characterization",
+                "screening",
+                "expression"
+            ]
+        },
+
+2. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version.
+
+**Specific example from the genetic modifications object upgrade:**
+
+For example if the original schema version for the genetic modification object being modified was "6", change it to "7" (6->7): 
         
         "schema_version": {
-            "default": "2"
+            "default": "7"
         }
 
-2. In the **upgrade** directory add an ```upgrade_step``` to an existing/new python file named after the object. An example upgrades the object from schema version (2->3):
+3. In the **upgrade** directory add an ```upgrade_step``` to a python file named after the object (create new if there is no such a file in the upgrade directory". For some objects the upgrade happens in the abstract class - like dataset for the experiment.
```

Comment:
In the **upgrade** directory add an ```upgrade_step``` to a python file named after the object (create new if there is no such a file in the upgrade directory). For some objects the upgrade is defined in the parent class - like dataset.py for the experiment object.